mk/external.mk

```diff
@@ -78,19 +89,31 @@ endef
 # Doom
 # https://tipsmake.com/how-to-run-doom-on-raspberry-pi-without-emulator
 DOOM_DATA_URL = http://www.doomworld.com/3ddownloads/ports/shareware_doom_iwad.zip
-DOOM_DATA = $(OUT)/DOOM1.WAD
+DOOM_DATA = $(DOOM_OUT)/DOOM1.WAD
 DOOM_DATA_SHA1 = 5b2e249b9c5133ec987b3ea77596381dc0d6bc1d
 
 # Quake
 QUAKE_DATA_URL = https://www.libsdl.org/projects/quake/data/quakesw-1.0.6.zip
-QUAKE_DATA = $(OUT)/id1/pak0.pak
+QUAKE_DATA = $(QUAKE_OUT)/id1/pak0.pak
```

Comment:
Ditto. Avoid specifying variables like `$(QUAKE_OUT)`.

Reply:
This is unwanted code. Reverted.